static/app/utils/dashboards/issueFieldRenderers.tsx

```diff
@@ -0,0 +1,259 @@
+  },
+  count: {
+    sortField: null,
+    renderFunc: data => issuesCountRenderer(data, 'count'),
+  },
+  userCount: {
+    sortField: null,
+    renderFunc: data => issuesCountRenderer(data, 'userCount'),
+  },
+  firstSeen: {
+    sortField: null,
+    renderFunc: ({firstSeen}) => <StyledDateTime date={firstSeen} />,
+  },
+  lastSeen: {
+    sortField: null,
+    renderFunc: ({lastSeen}) => <StyledDateTime date={lastSeen} />,
+  },
+};
+
+const issuesCountRenderer = (
+  data: EventData,
+  field: 'count' | 'userCount' | 'lifetimeCount' | 'lifetimeUserCount'
+) => {
+  const {selectionDateString} = data;
+  const isUserField = field.includes('user');
+  const primaryCount = data[field];
+  const count = data[isUserField ? 'userCount' : 'count'];
+  const lifetimeCount = data[isUserField ? 'lifetimeUserCount' : 'lifetimeCount'];
+  const filteredCount = data[isUserField ? 'filteredUserCount' : 'filteredCount'];
+  return (
+    <Container>
+      <Tooltip
+        isHoverable
+        skipWrapper
+        popperStyle={{padding: 0}}
+        title={
+          <div>
+            {filteredCount ? (
+              <React.Fragment>
+                <Item>
+                  <StyledContent>
+                    {t('Matching search filters')}
+                    <ItemCount value={filteredCount} />
+                  </StyledContent>
+                </Item>
+                <Divider />
+              </React.Fragment>
+            ) : null}
+            <Item>
+              <StyledContent>
+                {t(`Total in ${selectionDateString}`)}
+                <ItemCount value={count} />
+              </StyledContent>
+            </Item>
+            <Divider />
+            <Item>
+              <StyledContent>
+                {t('Since issue began')}
+                <ItemCount value={lifetimeCount} />
+              </StyledContent>
+            </Item>
+          </div>
+        }
+      >
+        <span>
+          {['count', 'userCount'].includes(field) && filteredCount ? (
+            <React.Fragment>
+              <Count value={filteredCount} />
+              <SecondaryCount value={primaryCount} />
+            </React.Fragment>
+          ) : (
+            <Count value={primaryCount} />
+          )}
+        </span>
+      </Tooltip>
+    </Container>
+  );
+};
+
+const Item = styled('li')`
+  list-style: none;
+`;
+
+const contentStyle = css`
+  width: 100%;
+  justify-content: space-between;
+  display: flex;
+  padding: 6px 10px;
+`;
+
+const StyledContent = styled('div')`
+  ${contentStyle};
+`;
+
+const SecondaryCount = styled(Count)`
+  :before {
+    content: '/';
+    padding-left: ${space(0.25)};
+    padding-right: 2px;
+  }
+`;
+
+const ItemCount = styled(({value, ...p}) => (
+  <div {...p}>
+    <Count value={value} />
+  </div>
+))`
+  text-align: right;
+  font-weight: bold;
+  font-variant-numeric: tabular-nums;
+  padding-left: ${space(2)};
+  color: ${p => p.theme.subText};
+`;
+
+const Divider = styled('li')`
```

Comment:
This can probably be a div, I think even for a11y it wouldn't provide any useful info

Reply:
Yup, updated to div!